### PR TITLE
debug log transaction sets

### PR DESCRIPTION
**Pull request by johnathanhowell**

### CODE REVIEW

File: modules/transactionpool/accept.go [hunk 1]
```diff
@@ -237,7 +237,18 @@ func (tp *TransactionPool) handleConflicts(ts []types.Transaction, conflicts []T
...
+	tp.transactionListSize += tsetSize
+
+	// debug logging
+	if build.DEBUG {
+		tp.log.Debugf("accepted transaction superset %v, size: %vB\n", setID, tsetSize)
+		for i, t := range superset {
+			tp.log.Debugf("superset transaction %v size: %vB\n", i, len(encoding.Marshal(t)))
+		}
+		tp.log.Debugf("tpool size is %vB after accepting transaction superset\n", tp.transactionListSize)
+	}
```
Comment (DavidVorick) [inline modules/transactionpool/accept.go]: This should all be one log. If you want multiple lines, use \n with \t\t\t for the next lines.

Comment (DavidVorick) [inline modules/transactionpool/accept.go]: Though, these logs aren't needed at all, only the other ones. Same comment to them though.

Comment (johnathanhowell) [inline modules/transactionpool/accept.go]: if these logs are removed, the transaction sets will never be logged if there is a conflict (acceptTransactionSet has the `if len(conflicts) > 0 return tp.handleConflicts..` branch)

Comment (DavidVorick) [inline modules/transactionpool/accept.go]: Oh I forgot it was structured that way. Ok, looks good then. But does still need to be a single log call, even if multiple lines.

**Pull request merged**